### PR TITLE
add translations for all texts

### PR DESCRIPTION
**Pull request by tflori**
this would solve #108 

@dregad what do you think? maybe you can provide french translations?

### CODE REVIEW

File: src/GetOpt.php [hunk 1]
```diff
@@ -384,29 +387,53 @@ public function setHelp(HelpInterface $help)
...
-        if (!$help instanceof Help) {
-            return false;
-        }
+        return self::setLang($language);
+    }
+
+    /**
+     * Translate $key
+     *
+     * Returns the key if no translation is found
```
Comment (dregad) [inline src/GetOpt.php]: Wouldn't it be better to fall back to the English translation by default ?

File: src/Translator.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+    public function __construct($language = 'en')
+    {
+        $this->setLanguage('en');
+    }
+
+    /**
+     * Translate $key
+     *
+     * Returns the key if no translation is found
```
Comment (dregad) [inline src/Translator.php]: Same comment as above

File: src/GetOpt.php [hunk 3]
```diff
@@ -399,7 +399,7 @@ public function setHelpLang($language = 'en')
     /**
      * Translate $key
      *
-     * Returns the key if no translation is found
+     * Returns english fallback or the key if no translation is found
```
Comment (dregad) [inline src/GetOpt.php]: Suggested rewording 

> Returns the translation for the given key; falls back to English if it is not localized in the configured language, and ultimately returns the key itself should it not exist in the English language file.

**Comment by dregad** [comment]
Not sure why the [comment I just posted](https://github.com/getopt-php/getopt-php/pull/116#pullrequestreview-110354253) is marked as _outdated_

File: src/GetOpt.php (hunk 3 above)
Comment (tflori) [reply src/GetOpt.php]: It was just too much text for me. But your are right: that is what the function is doing :smile:

Comment (dregad) [reply src/GetOpt.php]: Yeah, who cares about documentation anyway ? Just read the f***ing code, right ? :stuck_out_tongue_winking_eye:

Comment (dregad) [reply src/GetOpt.php]: See #117

**Pull request merged**